CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ internal API changes are not present.
 Main (unreleased)
 -----------------
 
+v0.36.0 (2023-08-29)
```

Comment:
I'd expect this to be an `-rc.1` version, [like in the 0.35 PR](https://github.com/grafana/agent/pull/4475/files). The data should be the date of the RC and not the one of the stable release.